### DISCUSSION TITLE
downgrade Apps

### DISCUSSION THREAD

**Discussion by jasonemperor** [xeon1989/Proton-X50-APK-Installer#40] [General]
I would like to downgrade from atlas to GKUI. Any idea? sir

**Comment by xeon1989** [comment]
@jasonemperor please refer to this issue for the firmware files https://github.com/xeon1989/Proton-X50-IHU-Upgrade-Downgrade/issues/1

**Reply by jasonemperor** [reply]
hi chris, thank you so much for replying me, i am too idiot for all these technical stuff. Really very, very thankful for your great help. Can I have your contact please? I need a master to guide me to downgrade it. I have attempted but fail, sir.

**Reply by jasonemperor** [reply]
Here's my personal email. jasonlam25@yahoo.com. , I truly hope you may reply me personally there.
